recipes-soletta/soletta/soletta_0.1.bb

```diff
@@ -4,7 +4,7 @@
 
 DESCRIPTION = "Soletta library and modules"
 SECTION = "examples"
-DEPENDS = "glib-2.0 libpcre pkgconfig"
+DEPENDS = "glib-2.0 libpcre pkgconfig python3-jsonschema-native icu curl"
```

Comment:
@barbieri, we used to ship python3-jsonschema to the image. Since Soletta just needs the host to have it, we've turned off the option.

The recipe exists (path: meta-soletta/recipes-modules/jsonschema) and can be installed by putting the following in the local.conf 
 ```IMAGE_INSTALL_append =  " python3-jsonschema"```